src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -65,18 +65,33 @@ struct OnDriverDistractionProcessor {
       policy::CheckPermissionResult result;
       application_manager_.GetPolicyHandler().CheckPermissions(
           application, stringified_function_id_, params, result);
+      const auto& msg_params =
+          (*on_driver_distraction_so_)[strings::msg_params];
+      const bool is_lock_screen_dissmisal_exists = msg_params.keyExists(
+          mobile_notification::lock_screen_dismissal_enabled);
+
+      if (is_lock_screen_dissmisal_exists &&
```

Comment:
`dissmisal` -> `dismissal`

Reply:
@jacobkeeler Thanks. Fixed in 35f6682680f94b26aa36f0b06385680c889776a6.